styles.css

```diff
@@ -1,81 +1,44 @@
-  quotes: none;
-}
-blockquote:before, blockquote:after,
-q:before, q:after {
-  content: '';
-  content: none;
-}
-table {
-  border-collapse: collapse;
-  border-spacing: 0;
+/* Generated by Font Squirrel (http://www.fontsquirrel.com) on April 6, 2015 and April 8, 2015 */
+@font-face {
+    font-family: 'yellowtailregular';
+    src: url('fonts/Yellowtail-Regular-webfont.eot');
+    src: url('fonts/Yellowtail-Regular-webfont.eot?#iefix') format('embedded-opentype'),
+         url('fonts/Yellowtail-Regular-webfont.woff2') format('woff2'),
+         url('fonts/Yellowtail-Regular-webfont.woff') format('woff'),
+         url('fonts/Yellowtail-Regular-webfont.ttf') format('truetype'),
+         url('fonts/Yellowtail-Regular-webfont.svg#yellowtailregular') format('svg');
+    font-weight: normal;
+    font-style: normal;
 }
-
-/* Generated by Font Squirrel (http://www.fontsquirrel.com) on April 6, 2015 */
 @font-face {
-    font-family: 'bebas_neuebold';
-    src: url('fonts/BebasNeue Bold-webfont.eot');
-    src: url('fonts/BebasNeue Bold-webfont.eot?#iefix') format('embedded-opentype'),
-         url('fonts/BebasNeue Bold-webfont.woff2') format('woff2'),
-         url('fonts/BebasNeue Bold-webfont.woff') format('woff'),
-         url('fonts/BebasNeue Bold-webfont.ttf') format('truetype'),
-         url('fonts/BebasNeue Bold-webfont.svg#bebas_neuebold') format('svg');
+    font-family: 'oswaldbold';
+    src: url('fonts/Oswald-Bold-webfont.eot');
+    src: url('fonts/Oswald-Bold-webfont.eot?#iefix') format('embedded-opentype'),
+         url('fonts/Oswald-Bold-webfont.woff2') format('woff2'),
+         url('fonts/Oswald-Bold-webfont.woff') format('woff'),
+         url('fonts/Oswald-Bold-webfont.ttf') format('truetype'),
+         url('fonts/Oswald-Bold-webfont.svg#oswaldbold') format('svg');
     font-weight: normal;
     font-style: normal;
 }
 @font-face {
-    font-family: 'architects_daughterregular';
-    src: url('fonts/ArchitectsDaughter-webfont.eot');
-    src: url('fonts/ArchitectsDaughter-webfont.eot?#iefix') format('embedded-opentype'),
-         url('fonts/ArchitectsDaughter-webfont.woff2') format('woff2'),
-         url('fonts/ArchitectsDaughter-webfont.woff') format('woff'),
-         url('fonts/ArchitectsDaughter-webfont.ttf') format('truetype'),
-         url('fonts/ArchitectsDaughter-webfont.svg#architects_daughterregular') format('svg');
+    font-family: 'patrick_handregular';
+    src: url('fonts/PatrickHand-Regular-webfont.eot');
+    src: url('fonts/PatrickHand-Regular-webfont.eot?#iefix') format('embedded-opentype'),
+         url('fonts/PatrickHand-Regular-webfont.woff2') format('woff2'),
+         url('fonts/PatrickHand-Regular-webfont.woff') format('woff'),
+         url('fonts/PatrickHand-Regular-webfont.ttf') format('truetype'),
+         url('fonts/PatrickHand-Regular-webfont.svg#patrick_handregular') format('svg');
     font-weight: normal;
     font-style: normal;
+
 }
 header, main, footer, .break{
   width: 900px;
```

Comment:
Try max-width here instead. This will show you how your page handles smaller width screens. You will likely need to change some of the other display properties (I think you may have an issue with absolute displayed elements).